docs/_articles/en/getting-started/java-setup.md

```diff
@@ -13,15 +13,14 @@ The Apex Language Server, shipped as part of the Salesforce Apex Extension for V
 
 Below you'll find instructions on how to download and configure different versions of Java.
 
-## AdoptOpenJDK
+## Adoptium
 
-[AdoptOpenJDK](https://adoptopenjdk.net/) provides prebuilt OpenJDK binaries for free without authentication or complications.
+[Adoptium](https://adoptopenjdk.net/) provides prebuilt OpenJDK binaries for free without authentication or complications.
```

Comment:
Do we want to point to adoptium.net link here?

Reply:
good catch. will update